wiki/osu!_Program_Files/User_Configuration_File/en.md

```diff
@@ -0,0 +1,232 @@
+# User Configuration File
+
+***[Click here to return to Main Page](/wiki/osu!_Program_Files)***
+
+![Combo Fire](/wiki/shared/combo-fire.jpg "Combo Fire")
```

Comment:
I don't think this image should be here anymore... the feature is long gone and if they want know some trivial things about it, they could just see the glossary